src/Command/Adapter/Csv.php

```
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Human Scoring Software
+ *
+ * @author antoinep@taotesting.com
+ * @license See LICENCE.md
+ */
+namespace App\Command\Adapter;
+
+use League\Csv\Reader;
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+
+class Csv implements AdapterInterface
+{
+    /**
+     * return an array of Scorer data from a csv file
+     *
+     * @param $filePath
+     * @return iterator
+     */
+    public function getIterator($filePath)
```

`\League\Csv\Exception` exception is not handled.